packages/connect-explorer/README.md

```diff
@@ -9,8 +9,8 @@ To run the explorer locally:
 
 `yarn workspace @trezor/connect-explorer dev` - runs dev server on port 8088
 
-When `connect-exporer` is run locally, WebUSB is disabled by Chrome because [@trezor/connect-popup](../connect-popup) runs on a different domain.
+When `connect-exporer` is run locally, WebUSB is disabled because [@trezor/connect-popup](../connect-popup) runs on a different domain and Chrome does not allow `navigator.usb.requestDevice` calls between cross-site elements since v72. Further investigation needed to see if we can fix that. Until then, run [Bridge](https://suite.trezor.io/web/bridge/) to connect to Trezor.
 
-Alternatively, use the [online explorer](https://trezor.github.io/connect-explorer/), which is pointing to the latest stable version of [@trezor/connect](../connect).
+Alternatively, use the [online explorer](https://trezor.github.io/trezor-suite/connect-explorer), which is pointing to the latest stable version of [@trezor/connect](../connect).
 
-For internal use, a build from the develop branch is available at [SLDEV](https://suite.corp.sldev.cz/connect-explorer/develop).
+For internal use, a build from the develop branch is available at [SLDEV](https://suite.corp.sldev.cz/trezor-suite/connect-explorer/develop).
```

Comment:
This one should remain `https://suite.corp.sldev.cz/connect-explorer/develop`, there is no /trezor-suite/connect-explorer/develop path on sldev